ethernet/w5500_spi/README.adoc

```diff
@@ -0,0 +1,95 @@
+= Attaching a W5500 via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to the W5500 chip. It is interfaced via SPI and power is supplied by the 3.3V pin.
+
+[NOTE]
+======
+This example simply sends and receives data from the W5500 chip using the polling approach. After you finish using the communication command* on the WIZ850io, ensure then the communication command set to clear.
+
+*The communication command is S"n"_CR (with "n" being the socket number).
+======
+
+== Importing external libraries
+
+In order to import the libraries needed for this example, you should first download all the files from the following respository: https://github.com/Wiznet/ioLibrary_Driver. Then in the CMakeLists.txt file in this directory replace "${YOUR_LIB_PATH_HERE}" on line 3 with the full path address of where the ioLibrary_driver folder is stored in your local machine.  
+
+The structure used to generally add additional libraries is:
```

Comment:
This section might need a bit of rewrite, to make it clearer what is generic and what is specific to this example.